public/index.html

```diff
@@ -48,24 +48,43 @@
     <body>
         <div class="centered">
             <h1>Twitodon</h1>
-            <h2>Find your Twitter friends on Mastodon</h2>
+            <h2>Add Your Twitter Following List to Your Mastodon Account</h2>
             <iframe src="https://github.com/sponsors/diddledani/button" title="Sponsor diddledani" height="35" width="116" style="border: 0;"></iframe>
         </div>
         <div>
-            <p>When you login to Twitodon with both your Twitter and Mastodon accounts it will memorise the association between your two accounts. Twitodon will then match any Twitter users you follow that have previously loged into Twitodon thereby providing their own association information. Unfortunately, there is no way of discovering, or guessing, Twitter users on Mastodon without them logging into Twitodon previously. Because of this, it is recommended to check back again regularly to keep up to date as more users use Twitodon.</p>
-            <p id="twitterLoginParagraph">Step 1. <a id="twitterLoginLink">Login With Twitter</a>. <button id="twitterLogoutLink" style="display: none">Logout and revoke access</button></p>
-            <p id="mastodonLoginParagraph">Step 2. <span id="mastodonLoginText">Login With Mastodon</span>. <button id="mastodonLogoutLink" style="display: none">Logout</button></p>
-            <p id="mastodonFormParagraph">Enter your Mastodon host's web address:</p>
+	    <h3>How Twitodon Works</h3>
+	    <p>Click the buttons below to give Twitodon access to your Twitter and Mastodon accounts. Don't worry, we won't be able to see your passwords - we use a secure, open standard called <a href="https://en.wikipedia.org/wiki/OAuth">OAuth</a> thatt provides a safe way for 3rd parties to access website information without divulging user passwords.</p>
+
+
+	    <p>Twitodon will cycle through your Twitter "following" list, checking to see if any of those accounts have also used this service and therefore linked their Twitter account to their Mastodon account. If they have, you'll get their Mastodon information in a nice file that you can upload. </p>
```

Comment:
- "cycle" suggests an unending repeating process
- Also a very long sentence - I've tried to suggest a more bitesize sentence

```suggestion
	    <p>Twitodon will scan your Twitter "following" list, checking to see if any of those accounts have also used this service. If they have used this service they will have linked their Twitter account to their Mastodon account, and you'll get their Mastodon information in a nice file that you can upload.</p>
```